src/classes/BDI_DataImportService.cls

```diff
@@ -99,21 +112,32 @@ global with sharing class BDI_DataImportService {
     * @return String the soql String
     */
     public static String strSoqlForBatchProcess(Id batchId) {
+        List<String> selectClause = new List<String>{
+            String.join(listStrDataImportFields, ','),
+            BATCH_NUMBER_QUERY_STR
+        };
+        String batchIdClause;
         String batchQuery;
```

Comment:
Same scenario here. Could we scrub the `batchQuery` string and return `new UTIL_Query()...`?

Reply:
fixed!